src/main.go

```diff
@@ -221,3 +227,24 @@ func main() {
 	log.Fatal(http.ListenAndServe(port, nil))
 
 }
+
+func setupLogger() {
+	currTime := time.Now()
+	currFileName := fmt.Sprint(logFilePrefix, currTime.Format("2006-01-02#15:04:05"), ".log")
+	user, err := user.Current()
+	if err != nil {
+		fmt.Printf("cannot access current user data\n")
+		return
+	}
+	homePath := user.HomeDir
+	logFilePath := homePath + "/" + currFileName
+	file, err := os.OpenFile(logFilePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if err != nil {
+		fmt.Printf("error opening log file : %s\n", logFilePath)
+		return
+	}
+	writer := io.MultiWriter(os.Stdout, file)
+	log.SetOutput(writer)
+	log.SetPrefix("LOG: ")
+	log.SetFlags(log.Ldate | log.Lmicroseconds | log.Llongfile)
```

Comment:
please use `log.Lshortfile`. long files log too much to find out the necessary incident.

Reply:
Done.